deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
+SCRIPTPATH=$(pwd -P)
+FILE1="docker-compose-base.yaml"
+FILE2="docker-compose-disk.yaml"
+FILE3="disk.env"
+
+#Prints instructions to show possible commands
+instructions() {
+	echo
+	echo "Usage:"
+	echo " To start the GAIA Hub type: $0 start."
+	echo " To stop the GAIA Hub type: $0 stop."
+	echo " To check if GAIA Hub is running type: $0 status."
+	echo " Simply typing $0 displays this help message."
+	echo
+}
+
+#Checks files I need exist
+check_files_exist() {
+	# If a file I need is missing, inform the user.
+	if ! test -f "$FILE1"; then 
+		echo "Missing $FILE1. Did you delete it?"
+		return 0
+	fi
+	if ! test -f "$FILE2"; then
+		echo "Missing $FILE2. Did you delete it?"
+		return 0
+	fi
+	if ! test -f "$FILE3"; then
+		echo "Missing $FILE3. Looks like you forgot to create one."
+		return 0
+	fi
+	# If all files I need exist, then continue
+	return 1
```

Comment:
In shell scripts, returning non-zero is conventionally considered an error.  The semantics of shell function return values are the same as those for `exit(2)`, since they are often executed within subshells or even child processes.  Returning 0 conventionally indicates success.  Can you update this?

Reply:
Ok. Done.